### PR TITLE
Home screen grid : increase max size

### PR DESCRIPTION
**Pull request by AirOne70**
Allow users to use a bigger grid. This is very useful on big screens or when icons size are reduced.

### CODE REVIEW

**Comment by suphon-t** [comment]
Allowing up to 15 makes it hard to select a value. Maybe reduce it to around 10?

**Comment by AirOne70** [comment]
12 steps may be too much yes. The old version of Lawnchair was allowing up to 20 rows / columns.
Limiting to 10 may not be enought depending on your DPI. I'm using 10 rows already on my personal setup.
On the other side, is a 3 rows setup really useful?

File: lawnchair/src/app/lawnchair/ui/preferences/HomeScreenPreferences.kt [hunk 1]
```diff
@@ -69,13 +69,13 @@ fun HomeScreenPreferences() {
                 label = stringResource(id = R.string.home_screen_columns),
                 adapter = prefs.workspaceColumns.getAdapter(),
                 step = 1f,
-                valueRange = 3.0F..7.0F
+                valueRange = 3.0F..15.0F
```
Comment (ghost) [inline lawnchair/src/app/lawnchair/ui/preferences/HomeScreenPreferences.kt]:
```suggestion
                valueRange = 3.0F..10.0F
```

File: lawnchair/src/app/lawnchair/ui/preferences/HomeScreenPreferences.kt [hunk 2]
```diff
@@ -69,13 +69,13 @@ fun HomeScreenPreferences() {
...
             )
             SliderPreference(
                 label = stringResource(id = R.string.home_screen_rows),
                 adapter = prefs.workspaceRows.getAdapter(),
                 step = 1f,
-                valueRange = 3.0F..7.0F,
+                valueRange = 3.0F..15.0F,
```
Comment (ghost) [inline lawnchair/src/app/lawnchair/ui/preferences/HomeScreenPreferences.kt]:
```suggestion
                valueRange = 3.0F..10.0F,
```

**Review by ghost — CHANGES_REQUESTED** [review]
As per @paphonb’s comment, I would suggest a maximum value of 10.

**Review by AirOne70 — COMMENTED** [review]
Changed to 10 as requested. :)

**Comment by ghost** [comment]
Cheers!

**Comment by RikKoedoot** [comment]
The appdrawer columns are not going up to 10, also if we are going with high numbers, is a slider still the best option?

**Comment by RikKoedoot** [comment]
Also dock icons can also only be changed up to 7

**Pull request merged**